examples/gno.land/r/demo/boards2/render.gno

```diff
@@ -20,29 +27,44 @@ func Render(path string) string {
 	return router.Render(path)
 }
 
-func renderBoardsList(res *mux.ResponseWriter, _ *mux.Request) {
+func renderBoardsList(res *mux.ResponseWriter, r request) {
 	res.Write("These are all the boards of this realm:\n\n")
-	gBoardsByID.Iterate("", "", func(_ string, value interface{}) bool {
-		board := value.(*Board)
+
+	pg := pager.NewPager(&gBoardsByID, defaultPageSize)
+	pageNum, limit, err := pg.ParseQuery(r.requestUri)
+	if err != nil {
+		panic(err)
+	}
+
+	page := pg.GetPageWithSize(pageNum, limit)
```

Comment:
@x1unix what do you think about just using `MustGetPageByPath` everywhere? Is there a constraint not to use it also here?

Reply:
@jeronimoalbi please check response above.